perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupAndGroupStructureSynchronizationIntegrationTest.java

```diff
@@ -495,6 +495,33 @@ public void modifyParentGroupTest() throws Exception {
 		assertGroup(testGroupB.getGroupName(), groupA.getId(), testGroupB.getDescription(), childOfBaseGroup);
 	}
 
+	@Test
+	public void mergeGroupAttributeTestList() throws Exception {
```

Comment:
Could you please also add a test for some attribute that should be rewritten?

Reply:
Added.